src/display/KT-LCD3/lcd.c

```diff
@@ -3413,226 +3413,142 @@ void lcd_print(uint32_t ui32_number, uint8_t ui8_lcd_field, uint8_t ui8_options)
 
 void lcd_enable_w_symbol (uint8_t ui8_state)
 {
-  if (ui8_state)
-    ui8_lcd_frame_buffer[9] |= 128;
-  else
-    ui8_lcd_frame_buffer[9] &= ~128;
+  if (ui8_state) { ui8_lcd_frame_buffer[9] |= 128; }
```

Comment:
Have you actually tested this change ?
I'm about 95% certain you don't intend to remove the else clause ;)
If you are dead-set on a one-liner, try this:
```
ui8_lcd_frame_buffer[9] = (ui8_lcd_frame_buffer[9] & ~128) | (ui8_state << 7)
```
(I'd actually be curious what ends up smaller in code size or faster, a compare + or / and, or a and + shift + or...)

Reply:
Yes of course and the way the firmware controls the LCD it is allowed :) It is such a small difference but every little bit matters.

No idea what ends up smaller or faster but I love those kind of questions! Do you have a test planned?